SoftLayer/CLI/routes.py

```diff
@@ -331,19 +331,17 @@
      'SoftLayer.CLI.securitygroup.interface:remove'),
     ('securitygroup:event-log', 'SoftLayer.CLI.securitygroup.event_log:get_by_request_id'),
 
-    ('sshkey', 'SoftLayer.CLI.sshkey'),
-    ('sshkey:add', 'SoftLayer.CLI.sshkey.add:cli'),
-    ('sshkey:remove', 'SoftLayer.CLI.sshkey.remove:cli'),
-    ('sshkey:edit', 'SoftLayer.CLI.sshkey.edit:cli'),
-    ('sshkey:list', 'SoftLayer.CLI.sshkey.list:cli'),
-    ('sshkey:print', 'SoftLayer.CLI.sshkey.print:cli'),
-
-    ('ssl', 'SoftLayer.CLI.ssl'),
-    ('ssl:add', 'SoftLayer.CLI.ssl.add:cli'),
-    ('ssl:download', 'SoftLayer.CLI.ssl.download:cli'),
-    ('ssl:edit', 'SoftLayer.CLI.ssl.edit:cli'),
-    ('ssl:list', 'SoftLayer.CLI.ssl.list:cli'),
-    ('ssl:remove', 'SoftLayer.CLI.ssl.remove:cli'),
```

Comment:
Add these back in, but pointed at the `SoftLayer.CLI.security.<command>` modules so that users can still call these commands. Thanks